code/modules/mob/living/simple_animal/animal_defense.dm

```diff
@@ -16,7 +16,23 @@
 		if("grab")
 			grabbedby(M)
 
-		if("harm", "disarm")
+		if("disarm")
+			M.do_attack_animation(src, ATTACK_EFFECT_DISARM)
+			playsound(src, 'sound/weapons/thudswoosh.ogg', 50, TRUE, -1)
+			var/shove_dir = get_dir(M, src)
+			if(!Move(get_step(src, shove_dir), shove_dir))
+				log_combat(M, src, "shoved", "failing to move it")
+				visible_message("<span class='danger'>[M.name] shoves [src]!</span>",
+					"<span class='danger'>You shove [src]!</span>", "<span class='hear'>You hear aggressive shuffling!</span>", COMBAT_MESSAGE_RANGE, src)
+				to_chat(src, "<span class='userdanger'>You're shoved by [M.name]!</span>")
+				return TRUE
+			log_combat(M, src, "shoved", "pushing it")
+			visible_message("<span class='danger'>[M.name] shoves [src], pushing [p_them()]!</span>",
+				"<span class='danger'>You shove [src], pushing [p_them()]!</span>", "<span class='hear'>You hear aggressive shuffling!</span>", COMBAT_MESSAGE_RANGE, src)
```

Comment:
Should this be runechatify'd

Reply:
There's currently no system for runechatifying combat messages.
I've discussed this with uomo before, and it would be interesting to add more classes of messages besides speech and emote into the runechat system, such as `warning` and `notice`.
`danger`, however, is not one of the things I'd runechatify because it could get spammy, it would block the view to things at the north of the target, and the action is already made visible by the attack animation and audible by the sound.

Reply:
I see, thank you for elaborating. I remember hearing something like that from uomo, but I wasn't quite sure on the details. I understand what you're worried about here.